osquery/tests/integration/tables/arp_cache.cpp

```diff
@@ -19,23 +19,15 @@ namespace osquery {
 class arpCache : public IntegrationTableTest {};
 
 TEST_F(arpCache, test_sanity) {
```

Comment:
May I ask you to change the name of the test class according to style guide to smt like `ArpCacheTest` ?